src/vdaf.rs

```diff
@@ -191,12 +194,13 @@ where
     /// method returns [`PrepareTransition::Finish`], at which point the returned output share may
     /// be aggregated. If the method returns an error, the aggregator should consider its input
     /// share invalid and not attempt to process it any further.
+    #[allow(clippy::type_complexity)]
     fn prepare_step(
```

Comment:
The docs for `prepare_init()` and `prepare_step()` should be revised to refer to `PrepareShare`